From 746150dec2665c8d3f783dbc634c8eaae26c1a29 Mon Sep 17 00:00:00 2001
From: Jing Leng <lengjingzju@163.com>
Date: Sun, 10 Sep 2023 20:47:58 +0800
Subject: [PATCH] alas-utils: Fix compile failure with glibc-2.38

Signed-off-by: Jing Leng <lengjingzju@163.com>
---
 alsactl/init_sysdeps.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/alsactl/init_sysdeps.c b/alsactl/init_sysdeps.c
index 3aca1b4..c8fa0b7 100644
--- a/alsactl/init_sysdeps.c
+++ b/alsactl/init_sysdeps.c
@@ -18,6 +18,7 @@
  */
 
 #if defined(__GLIBC__) && !(defined(__UCLIBC__) && defined(__USE_BSD))
+#if (__GLIBC__ < 2) || ((__GLIBC__ == 2) && (__GLIBC_MINOR__ < 38))
 static size_t strlcpy(char *dst, const char *src, size_t size)
 {
 	size_t bytes = 0;
@@ -60,4 +61,5 @@ static size_t strlcat(char *dst, const char *src, size_t size)
 	*q = '\0';
 	return bytes;
 }
+#endif
 #endif /* __GLIBC__ */
-- 
2.25.1

